Kernel/Bus/USB/xHCI/xHCIController.cpp

```diff
@@ -867,7 +867,8 @@ ErrorOr<size_t> xHCIController::submit_bulk_transfer(Transfer& transfer)
     pending_transfer.wait_queue.wait_forever();
     VERIFY(!pending_transfer.endpoint_list_node.is_in_list());
 
-    if (pending_transfer.completion_code != TransferRequestBlock::CompletionCode::Success)
+    if (pending_transfer.completion_code != TransferRequestBlock::CompletionCode::Success
```

Comment:
commit message nit: "fewer bytes", not "less bytes". ("fewer" for countable things, "less" for uncountable things. bytes are countable.)